.github/workflows/ci.yml

```diff
@@ -99,7 +99,6 @@ jobs:
             happy_path_restart_alice_after_xmr_locked,
             alice_and_bob_refund_using_cancel_and_refund_command,
             alice_and_bob_refund_using_cancel_and_refund_command_timelock_not_expired,
-            alice_and_bob_refund_using_cancel_and_refund_command_timelock_not_expired_force,
```

Comment:
You need to delete it from the `bors.toml` as well: https://github.com/comit-network/xmr-btc-swap/blob/7be6bcd4f5747bcce5ddb9d628eebe4d5da045ab/bors.toml#L15

Reply:
Otherwise bors is going to timeout because it will wait for the status forever.